js/app/controllers/calcontroller.js

```diff
@@ -140,15 +140,18 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
+			constants.publicSharingToken.split(".").forEach(token=>
```

Comment:
Add spaces before and after token, and I think it should be inside some parenthesis.